src/Helmet.jsx

```diff
@@ -29,6 +29,11 @@ const getInnermostProperty = (propsList, property) => {
     return null;
 };
 
+const getInnermostCallback = (propsList, property) => {
+    const deepestProps = [...propsList].reverse()[0];
+    return deepestProps ? deepestProps[property] : null;
+};
+
```

Comment:
This works perfect, but maybe we can rename it to make it more reusable.  `getInnermost` has the same prefix of `getInnermostProperty` which may confuse people thinking they do the same thing.  And having `callback` in the name suggests it can only be used for callbacks.  What do you think about something with property, to make it more general - `getStrictInnermostProperty`?  Ahh, naming is so hard.  Open to suggestions.